doc/modules/preprocessing.rst

```diff
@@ -299,8 +302,42 @@ This can be confirmed on a independent testing set with similar remarks::
   ... # doctest: +ELLIPSIS +SKIP
   array([ 0.01...,  0.25...,  0.46...,  0.60... ,  0.94...])
 
-It is also possible to map the transformed data to a normal distribution by
-setting ``output_distribution='normal'``::
+Mapping to a Gaussian distribution
+----------------------------------
+
+In many modeling scenarios, normality of the features in a dataset is desirable.
+Power transforms are a family of parameteric transformations that aim to
+map data from any distribution to as close to a Gaussian distribution as
+possible.
```

Comment:
You should mention it only works for positive data.

Reply:
We should probably say that this is optimized to minimize skewedness? do we want the paper in the user guide as well maybe?